packages/expo-image/ios/EXImageView.m

```diff
@@ -43,9 +85,33 @@ - (void)updateImage
     self.onLoadStart(@{});
   }
 
-  [self sd_setImageWithURL:_imageURL
+  NSURL *imageURL = _source ? [RCTConvert NSURL:_source[sourceUriKey]] : nil;
+  NSNumber *scale = _source && _source[sourceScaleKey] ? _source[sourceScaleKey] : nil;
```

Comment:
```suggestion
  NSNumber *scale = _source && _source[sourceScaleKey] ?: nil;
```
Could be written shorter with the Elvis operator

Reply:
Actually, your suggestion doesn't compile. I was able to write it shorter into

`NSNumber *scale = _source ? _source[sourceScaleKey] : nil;`

Reply:
Ah, ok, sorry, didn't notice that the fallback value is `nil`. If so, I think we could just use

```objc
NSNumber *scale = _source[sourceScaleKey];
```

I ensured this should be ok by running

```objc
  NSDictionary *dict = nil;
  NSLog(@"%@", dict[@"abc"]);
```
which didn't crash the application (logged `(null)`).

Reply:
Works like a charm. Must be how Objective-C sends messages to objects that makes this work probably.

Updated the code, looks a lot slimmer 👍